rio_tiler/models.py

```diff
@@ -155,10 +155,27 @@ def transform(self):
     def post_process(
         self,
         in_range: Optional[Tuple[NumType, NumType]] = None,
-        out_range: Optional[Tuple[NumType, NumType]] = None,
+        out_dtype: str = "uint8",
```

Comment:
For typing I think you should allow more than just `str`. I.e. I prefer to use `np.uint8` for typing, and that would fail here. So I'd suggest

```suggestion
        out_dtype: Union[str, numpy.number] = "uint8",
```

`np.number` should cover all float and integer types